tools/Makefile.common

```diff
@@ -8,7 +8,7 @@ BP_TOOLS_INSTALL_DIR := $(BP_TOOLS_DIR)/install
 BP_TOOLS_BIN_DIR     := $(BP_TOOLS_INSTALL_DIR)/bin
 BP_TOOLS_LIB_DIR     := $(BP_TOOLS_INSTALL_DIR)/lib
 BP_TOOLS_INCLUDE_DIR := $(BP_TOOLS_INSTALL_DIR)/include
-BP_TOOLS_TOUCH_DIR   := $(BP_TOOLS_INSTALL_DIR)/touchfiles/tools
+BP_TOOLS_TOUCH_DIR   := $(BP_TOOLS_INSTALL_DIR)/touchfiles
```

Comment:
keep this?

Reply:
Yes. The touchfiles directory is already under TOOLS_INSTALL, so there is no need to create this extra subdirectory. I will double check the tools building to confirm, but I noticed it when building AXE.

Reply:
Sure that makes sense, but the base install path for tools is `rtl/tools/install`. Then, the tools' touchfiles directory is a subdirectory of that. It is redundant to have `rtl/tools/install/touchfiles/tools`. If we had some progs or etc things that were building under rtl they shouldn't install into `rtl/tools/install`. Everything should install into a generic `rtl/install` directory like the SDK.

Reply:
This change fixes the redundant directory. It does not address the fact that a top-level `rtl/install` directory does not exist/is not used (as the SDK does)